src/Bridge/Symfony/Routing/ApiLoader.php

```diff
@@ -71,21 +79,96 @@ public function load($data, $type = null)
 
             if (null !== $collectionOperations = $resourceMetadata->getCollectionOperations()) {
                 foreach ($collectionOperations as $operationName => $operation) {
-                    $this->addRoute($routeCollection, $resourceClass, $operationName, $operation, $resourceShortName, true);
+                    $this->addRoute($routeCollection, $resourceClass, $operationName, $operation, $resourceShortName, OperationTypes::COLLECTION);
                 }
             }
 
             if (null !== $itemOperations = $resourceMetadata->getItemOperations()) {
                 foreach ($itemOperations as $operationName => $operation) {
-                    $this->addRoute($routeCollection, $resourceClass, $operationName, $operation, $resourceShortName, false);
+                    $this->addRoute($routeCollection, $resourceClass, $operationName, $operation, $resourceShortName, OperationTypes::ITEM);
                 }
             }
+
+            $this->computeSubcollectionOperations($routeCollection, $resourceClass);
         }
 
         return $routeCollection;
     }
 
     /**
+     * Handles subcollection operations recursively and declare their corresponding routes.
+     *
+     * @param RouteCollection $routeCollection
+     * @param string          $resourceClass
+     * @param string          $rootResourceClass null on the first iteration, it then keeps track of the origin resource class
+     * @param string          $rootResourceClass null on the first iteration, it then keeps track of the parent operation
+     */
+    private function computeSubcollectionOperations(RouteCollection $routeCollection, string $resourceClass, $rootResourceClass = null, $parentOperation = null)
+    {
+        if (null === $rootResourceClass) {
+            $rootResourceClass = $resourceClass;
+        }
+
+        foreach ($this->propertyNameCollectionFactory->create($resourceClass) as $property) {
+            $propertyMetadata = $this->propertyMetadataFactory->create($resourceClass, $property);
+
+            if (!$propertyMetadata->hasSubcollection()) {
+                continue;
+            }
+
+            $subcollection = $propertyMetadata->getType()->isCollection() ? $propertyMetadata->getType()->getCollectionValueType()->getClassName() : $propertyMetadata->getType()->getClassName();
+
+            $propertyName = Inflector::pluralize(Inflector::tableize($property));
+
+            $operation = [
+                'property' => $property,
+            ];
+
+            if (null === $parentOperation) {
+                $rootResourceMetadata = $this->resourceMetadataFactory->create($rootResourceClass);
+                $rootShortname = $rootResourceMetadata->getShortName();
+                $resourceRouteName = Inflector::pluralize(Inflector::tableize($rootShortname));
```

Comment:
Sorry about that, I was sure it was configurable :p. Maybe can you group all those calls in a private method to be able to make the strategy configurable later if we want to.